variables.tf

```diff
@@ -64,3 +64,14 @@ variable "chmod_command" {
   default     = "chmod 600 %v"
   description = "Template of the command executed on the private key file"
 }
+
+variable "region" {
+  type        = "string"
+  default     = "us-east-1"
```

Comment:
You mean so: `variable "region" {}` ?

Reply:
You are right, it is better to make a new key generation module. Where to apply pull request?

Reply:
Yea, so in that case we should create `terraform-tls-key-pair` since it's based on the terraform `tls` provider: https://www.terraform.io/docs/providers/tls/index.html

I can create the repo and you can submit the pr against it.

Reply:
Here's the new repo: https://github.com/cloudposse/terraform-tls-key-pair